README.md

```diff
@@ -360,10 +360,40 @@ nginx_http_template:
             weight: 1
             health_check: max_fails=1 fail_timeout=10s
 
-# Enable creating dynamic templated NGINX stream configuration files.
+# Enable creating dynamic templated NGINX Stream configuration files.
```

Comment:
I'd tweak this description, your network_stream block (aka reverse proxying block) is compatible with having upstreams, and the stream module does not support web servers. Stream use cases are going to be limited for the most part to reverse proxying / load balancing.

Reply:
That was a miss on my part.  Does this sound ok?

# Comment out upstreams depending on whether you wish to create a reverse proxy
# or load balancer configuration file.

Reply:
That could work, although I would delete that whole sentence since in the end even if you use upstreams you are load balancing by acting as a reverse proxy. The only difference between using upstreams or not is whether you want to load balance or not.